scripts/make_docs.sh

```diff
@@ -4,9 +4,6 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-# Exit if any error occurs
-set -e
-
```

Comment:
This might not be ideal since we run the tutorials against latest as part of this script -- which I am not a fan of --